src/krux/pages/settings_page.py

```diff
@@ -355,11 +357,17 @@ def category_setting(self, settings_namespace, setting):
                     DEFAULT_PADDING,
                     t("Right"),
                 )
-            self.ctx.display.draw_centered_text(
-                settings_namespace.label(setting.attr) + "\n" + str(current_category),
+            lines = self.ctx.display.draw_centered_text(
+                "\n\n" + str(current_category),
```

Comment:
Please use offset instead of "\n" to change position of text

Reply:
This call is for `draw_centered_text`, it is more complicated as it calculates the offset based on the text size in lines and display height... I think if I try to optimize this, it will turn out to be slower

Reply:
Since drawing was separated, the best then is to change to `draw_hcentered_text`

Reply:
Changed category settings to use `draw_hcentered_text` :+1: